### ISSUE TITLE
Use gophp7.org

### ISSUE THREAD

**Issue by salathe** [TomasVotruba/gophp71.org#7]
Let's use gophp7.org and monitor timelines for packages requiring 7.0, 7.1, 7.2, ...

**Comment by TomasVotruba** [comment]
Hey @salathe,
what exactly do you mean? Is that your project?

**Comment by salathe** [comment]
> what exactly do you mean?

I was just thinking, why restrict it to 7.1 only?

> Is that your project?

Yes

**Comment by TomasVotruba** [comment]
**tl;dr;** If Symfony would go to 7.1 and Doctrine to 7.0, which PHP version you'd need to use them both? 

---

Well the point of all this is not a statistics (to monitor various versions for various promentiojects - *monitor timelines for packages requiring 7.0, 7.1, 7.2, ...*).

**But to unite and bump version together.**

Have you read my post about this topic? https://www.tomasvotruba.cz/blog/2017/06/05/go-php-71/

I quote [PHP 5 movement](https://www.garfieldtech.com/blog/go-php-5-go) that actually inspired me to it:

> A while back, various people were lamenting the state of PHP 5 adoption, myself included. What to do about it? How to get hosts to let programmers leverage the added functionality that PHP 5 offers? How can we do that without cutting off 80% of our user base?

> The solution a few people suggested was team work. If all PHP projects stopped supporting PHP 4 and made the jump to PHP 5 at the same time, none of them is penalized in the market for being "first" and web hosts will have a clear business case to upgrade their systems to PHP 5. We can then all start offering faster, cleaner, more powerful, more secure web software.

The idea was about hostings, but the same goes for open-source projects, as in **tl;dr;**

**Comment by salathe** [comment]
I understand your desire to focus on getting people to one specific version of PHP 7.  The offer of using gophp7.org is still there.

**Comment by TomasVotruba** [comment]
Well it would be great if you could redirect it to https://gophp71.org and promote it.

**Comment by TomasVotruba** [comment]
Any updates?

**Comment by TomasVotruba** [comment]
Closing for no response. Feel free to reopen with one

**Comment by salathe** [comment]
Thanks for closing this out.
